arrays-and-scope.js

```diff
@@ -1,25 +1,32 @@
-
 const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // --------------------------
 // Opdracht 1: Check hoeveel studenten hun studie cum laude hebben gehaald (8 of hoger) en log dit in de console.
 // verwacht antwoord: 6
 // --------------------------
+let cumLaude = 0;
 
+for (let i = 0; i < grades.length; i++) {
```

Comment:
Deze manier werkt uiteraard net zo goed, maar je zou hier ook een for-of loop kunnen gebruiken:

```
For (Const grade of grades) {
        if (grades[i] >= 8) {
             cumLaude++;
        }
}